backend/api/Areas/Notes/Models/NoteModel.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace Pims.Api.Areas.Notes.Models
```

Comment:
Models under the Areas are not re-usable. If possible please move them to the Concepts folder and namespace

Reply:
Made the changes, please have a look